client/assets/translations/base.pot

```diff
@@ -102,23 +89,10 @@ msgstr ""
 msgid "CAPTURE"
 msgstr ""
 
-#: levels/game.tscn
-#: logic/ui/floating_info.tscn
-msgid "100"
-msgstr ""
-
-#: levels/game.tscn
-msgid "2\n"
-msgstr ""
-
 #: levels/game.tscn
 msgid "NEXT_ROUND"
 msgstr ""
 
-#: levels/game.tscn
-msgid "Player 1 Turn"
```

Comment:
I'm not sure if this one should be dropped or transformed into a msgId. 

Also, maybe makes sense to have something like "Player {{playerNum}} Turn" that can be translated in spanish like "Turno del juegador {{playerNum}}"

Reply:
Agree. Have you tested it in the game? Does the turn announcement still work?

Reply:
this is not needed, with the current improvements with {player_name}, it shows like this:
![image](https://github.com/user-attachments/assets/50800ff7-b72c-40ea-9e80-a64616c91d0d)